microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -281,16 +302,17 @@ There are several ways to open the extension in the sidebar:
 
 For more Sidebar API extensions demos, explore any of the following extensions:
 
-* [Site-specific side panel example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-site-specific)
-* [Multiple side panels example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-multiple)
-* [Global side panel example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-global)
 * [Dictionary side panel example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/sample.sidepanel-dictionary)
+* [Global side panel example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-global)
+* [Multiple side panels example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-multiple)
+* [Opening the side panel through a user interaction](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-open)
```

Comment:
note 5th sample link added in Chrome docs https://developer.chrome.com/docs/extensions/reference/sidePanel/#examples -- 
* [Opening the side panel through a user interaction](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-open)

Reply:
That sample requires the sidePanel.open API, which Chrome is rolling out in 116 and we will in 117.

Reply:
```
<!--
todo: un-comment and place as list item 4 of 5:
* [Opening the side panel through a user interaction](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-open)
this sample requires sidePanel.open API, which Edge will roll out in Edge Stable 117.  As of August 18, Edge Stable = 115, Canary = 118.
-->
```